webapp/app/css/treeherder.css

```diff
@@ -1035,6 +1046,7 @@ ul.failure-summary-list li .btn-xs {
 
 .pinned-job-close-btn {
     padding: 1px 2px 1px 2px;
+    border-color: #fafafa #fafafa #fafafa transparent;
```

Comment:
afaik shorthand hex only works for codes where the R, G or B component has the same letter. So #FFCC00 can be shortened to #fc0, but #fa is not valid.